login.css

```diff
@@ -61,6 +61,21 @@ ul li ul{
 	width: 100px;
 	display: none;
 	visibility: hidden;
+
+}
+	
+ul li ul li{
```

Comment:
05/08.2021: Apply the drop down menu to make the nav bar to be more clean. So I was apply the drop down menu so that it can hide some link so it doesn't make the navbar look messy with a lot of link. which can make the aesthetics of the music database to be more professional. Also at a hover so that when the user hover a link then it will change the background to back and text color to white which is display a great contrast so the user can focus on the link that they are hover on.